### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by wpsilvestre** [jasp-stats/jasp-issues#2689]
### Description

Adding, SEM, SD, and CV to ANOVA descriptive statistics table

### Purpose

Adding the standard error of the mean, standard deviation, and coefficient of variation to ANOVA descriptive statistics

### Use-case

This feature would benefit any user that works with Jasp

### Is your feature request related to a problem?

No

### Is your feature request related to a JASP module?

ANOVA

### Describe the solution you would like

Having the option of showing the standard error of the mean, standard deviation, and coefficient of variation to ANOVA descriptive statistics table

### Describe alternatives that you have considered

I tried using the 'descriptive statistics' module, but these parameters would be related to the ANOVA analysis, rather than being conducted separately, which makes the software used truncated.

### Additional context

Showing this information (SEM, SD, and CV to ANOVA descriptive statistics table) would ease the interpretation of the results and helping users to add meaningful data to the results of the computed statsitics with Jasp.

**Comment by tomtomme** [comment]
Interesting. I teach ANOVA since >10 years and never thought about this one. Can you explain why the non-split information about the AV is relevant in the ANOVA context? I am just curious.

**Comment by JohnnyDoorn** [comment]
@wpsilvestre can you give a concrete example? I would also think that if you want descriptives split by your factors, you can use the descriptive statistics option in ANOVA, and if you want the non-split descriptives you can use the descriptive module.

**Comment by wpsilvestre** [comment]
Dear @tomtomme, 
As far as I know, having this information (SEM, SD, and, especially, the CV) helps understand the variability of the dataset analysis and may be an interesting parameter to show with the anova table. But, please, correct me if I'm wrong.

@JohnnyDoorn, I am aware of the descriptive module, but having these values automatically calculated with the respective dataset may be an interesting addition to the anova module. For example, if I want to know the CV associated with the analysis, having this option with the anova table (or a general descriptive) would be interesting (and without the need to access the descriptive statistics module and risk choosing a wrong variable/dataset).
Currently, the anova module shows the SEM, SD, and CV of each level in the descriptive table. I believe that bringing this information for the whole analysis would help the people using Jasp.

I hope I was clear. I am at your disposal to discuss and learn more. Thanks.

**Comment by wpsilvestre** [comment]
@tomtomme, I agree with all you said, but if the dataset has a large CV, this means the data has a wide variability (and I am not talking about homoscedasticity or residuals). Data with large CV may mask treatment effects, and assessing this parameter may help the readers (and who is analyzing the data) to assess whether the data has a large variation (and more samples/runs could/would be performed, or even the quality of the analysis), or it is within a reasonable variation. Am I wrong in my assumption here?

**Comment by wpsilvestre** [comment]
@tomtomme, I agree with all you said, but if the dataset has a large CV, this means it has a wide variability (and I am not talking about homoscedasticity or residuals). Data with a large CV may mask treatment effects, and assessing this parameter may help the readers (and who is analyzing the data) to assess whether the data has a large variation, or it is within a reasonable range, even whether the quality of the analysis. Maybe am I wrong with may assumption here?

**Comment by tomtomme** [comment]
There we have it! Sounds like a logic assessment to me! Thx for the insight. I never thought about it this way.

**Comment by wpsilvestre** [comment]
@tomtomme, I appreciate the patience and I'm happy to help. Would it be feasible to implement such a feature (and the option to have it or not shown in the results) in the anova module of Jasp?

**Comment by tomtomme** [comment]
I hope so. But I can give no timeline.

**Comment by JohnnyDoorn** [comment]
Hi @wpsilvestre,

I'm a bit hesitant to add another checkbox for descriptives, since we already provide descriptives for each level of the predictors, and if the user wants overall statistics for the DV this is straightforward to do through the descriptives module. One important thing we need to look out for is to not clutter the interface too much with redundant options. For instance, we also offer Q-Q plots of the DV in descriptives, but only Q-Q plots of the residuals of the model in ANOVA (and not Q-Q of the DV). So I would be highly in favor of having mores analysis-specific output in the ANOVA (such as factor level descriptives), while keeping the general descriptives stuff in descriptives (such as variable level descriptives). 

Kind regards
Johnny

**Comment by tomtomme** [comment]
@JohnnyDoorn 
Yes, this should definitely not be implemented via another checkbox. But we could just add an "overall" or "total" row to the descriptives table by default, similar to the "total" row in contingency tables. 
But I can understand if you still do not want to duplicate this stuff, since one can get it from descriptives.

**Comment by JohnnyDoorn** [comment]
That's a great suggestion @tomtomme, I'll look into it!

**Comment by wpsilvestre** [comment]
@JohnnyDoorn  and @tomtomme , thanks for the feedback! The idea @tomtomme brought is excellent.

I have another doubt/request regarding the post hoc tests of anova. I can explain here, or would it be better to open a new topic?

Thanks again for the attention.

**Comment by JohnnyDoorn** [comment]
@wpsilvestre I think it's best to make a separate feature request or bug report for the post hoc test - please tag me in it, then i can take a look immediately!

**Comment by tomtomme** [comment]
> @JohnnyDoorn Yes, this should definitely not be implemented via another checkbox. But we could just add an "overall" or "total" row to the descriptives table by default, similar to the "total" row in contingency tables. But I can understand if you still do not want to duplicate this stuff, since one can get it from descriptives.

I just saw this usefull difference line for the dep t-Test:
![20240518_122953](https://github.com/jasp-stats/jasp-issues/assets/4428185/f3d20cb4-e90d-478d-806f-2569ecf67928)

So I guess we should implement this for descriptives tables of all group comparisons
- independent t-Test & ANOVA & RM ANOVA: total row
- one sample wilcox & z & t & all 3 t-Test: difference row
